openedx/core/djangoapps/profile_images/views.py

```diff
@@ -84,8 +97,8 @@ def post(self, request, username):
                 # generate profile pic and thumbnails and store them
                 create_profile_images(uploaded_file, get_profile_image_names(username))
```

Comment:
get_profile_image_names will also call _make_image_version. We should not call _make_image_version twice because it could potentially return different things. How about get_profile_image_names also returns the result of _make_image_version?